### PR TITLE
Adding footer logo

### PR DESCRIPTION
**Pull request by Peartes**
We had originally removed the Prime Dao logo from the footer which is an undesied change, this commit adds the logo back and closes this [issue](https://www.notion.so/curvelabs/339a249185e14ddcbd731e5ba07c46c7?v=f7cba822bb9043a5a87ab80b5e2b40ef&p=f16f086f74274cedb89f752a41eefd8c)

### CODE REVIEW

File: src/footer/footer.html [hunk 1]
```diff
@@ -53,6 +53,7 @@
 
     <div class="bottom">
       <div class="copyright"><img src="/cc.svg">2021</div>
```
Comment (dkent600) [inline src/footer/footer.html:55]: I don't see 2021 in the screen shot.  I also don't see the copyright icon.  Aren't you supposed to get rid of those?

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/footer/footer.html (hunk 1 above)
Comment (Peartes) [reply src/footer/footer.html:55]: Sorry, I don't understand. What screenshot? The 2021 icon and copyright icon are still there, I did not get rid of them.

Comment (dkent600) [reply src/footer/footer.html:55]: The screenshot in the notion card that shows what this PR is supposed to be doing

**Review by dkent600 — CHANGES_REQUESTED** [review]
Isn't looking right on small widths.  (c) is gone, "Governed By" is still visible, and the text is scrunched.

image

**Review by dkent600 — CHANGES_REQUESTED** [review]
logo is missing at wide widths

image

**Review by dkent600 — CHANGES_REQUESTED** [review]
I think this will look a lot better, and Pat will like it a lot better, if we make the (c) Prime Dev... text be centered above a line that contains only Privacy Policy and Cookies.  This would only be needed at the widths where the logo moves to the top and  Privacy Policy and Cookies move to the bottom.  Can you please make that change?

**Review by dkent600 — APPROVED** [review]

**Pull request merged**